src/dmd/opover.d

```diff
@@ -1038,6 +1038,7 @@ Expression op_overload(Expression e, Scope* sc)
                             // Rewrite (e1 op e2) as e2.opfunc(e1)
                             result = build_overload(e.loc, sc, e.e2, e.e1, m.lastf ? m.lastf : s);
                         }
+/+
```

Comment:
What's up with this?

Reply:
I believe it is dead code. So I'm testing it out.

Reply:
Turns out it's not dead, it some wacky signalling device to the caller. I'll redo that in another PR.